ramls/item.json

```diff
@@ -128,6 +132,34 @@
             "type": "string",
             "description": "Text to display"
           },
+          "source": {
+            "type": "object",
+            "description": "The user who added the note",
+            "properties": {
```

Comment:
As these properties are intended to be owned by the server, I think it could be useful to reflect that in the description

Reply:
Updated descriptions for the user and the date.